package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "webtorrent-desktop",
   "description": "WebTorrent, the streaming torrent client. For Mac, Windows, and Linux.",
-  "version": "0.19.0",
+  "version": "0.20.0",
```

Comment:
The version will be incremented during the release process, so I'll undo this change in this PR!